### ISSUE TITLE
Make Table of contents sticky

### ISSUE THREAD

**Issue by Serpentarius13** [nodejs/nodejs.org#7122]
### Enter your suggestions in details:

1. I want Table of contents to be sticky
2. It would be easier to have an overview at all places of the documentation page
3. It should be quite easy to accomplish and slap position: sticky to it. I could submit a PR

**Comment by AugustinMauroy** [comment]
What are you talking about ? Which layout ?

**Comment by Serpentarius13** [comment]
> What are you talking about ? Which layout ?

This one 
![image](https://github.com/user-attachments/assets/3b4a339e-44a8-41d7-a2cc-b83eb4396ab0)

**Comment by AugustinMauroy** [comment]
Okay I'm +1. But need to sticky all the metabar IMO

**Comment by Serpentarius13** [comment]
> Okay I'm +1. But need to sticky all the metabar IMO

Cool! But why include everything else? When reading it is not really useful info, no?

**Comment by AugustinMauroy** [comment]
> But why include everything else? When reading it is not really useful info, no?

IMO, To keep a consistent look with the entire metabar. Then you have to test to see what looks best
